internal/io/http/client.go

```diff
@@ -116,7 +116,7 @@ func (cc *ClientConf) InitConf(device string, props map[string]interface{}) erro
 	default:
 		return fmt.Errorf("Not supported HTTP method %s.", c.Method)
 	}
-	if c.Interval <= 0 {
+	if needInterval && c.Interval <= 0 {
```

Comment:
RestSink does not need interval too. There must be some mechanism to bypass this check. Maybe we can reuse that instead of add a new property